runtime/src/lib.rs

```diff
@@ -1007,7 +1009,7 @@ impl pallet_ismp::Config for Runtime {
 
 impl pallet_hyperbridge_aggregations::Config for Runtime {
     type RuntimeEvent = RuntimeEvent;
-    type IsmpDispatcher = pallet_ismp::Pallet<Runtime>;
+    type IsmpDispatcher = pallet_hyperbridge::Pallet<Runtime>;
```

Comment:
Is this change needed? What are the consequences?

Reply:
yes this change is needed, Actually the ISMP dispatcher will be used to dispatch cross-chain requests. Pallet hyperbridge provides an IsmpDispatcher implementation that handles both protocol and relayer fee collection.
So actually this is the way to say that we are actually using pallet hyperbridge in the whole flow and fee will be collected.